app/docs/md/learn/practices/testing.md

```diff
@@ -85,3 +85,40 @@ test('Shut down local server', async t => {
 })
 ```
 
+## Component testing
+
+If you'd like to run functional component tests, we recommend using [WebdriverIO](https://webdriver.io/) as a browser automation framework. The benefits of using an actual browser compared to JSDOM outweigh the minimal speed detriment that comes with spinning up a browser instance. Furthermore, WebdriverIO is based on [WebDriver](https://www.w3.org/TR/webdriver/), a browser automation web standard which guarantees that your tests interact with your component as closely as possible to how a user would.
```

Comment:
Re-adding the second part of [my previous comment](https://github.com/enhance-dev/enhance.dev/pull/132#discussion_r1099167782) for visibility, since the suggested changes in the first part were applied, which marks the thread as resolved:

> Overall, I feel that this passage is lacking a bit of context. IMO the second sentence should be prefaced by explicitly mentioning that other similar testing frameworks use JSDOM (and at least a link to a page defining what JSDOM is), and that WebDriver uses an actual browser instead.

Reply:
Indeed the [new text](https://github.com/enhance-dev/enhance.dev/pull/132/commits/f7665518aacfe19731f23239c8cc8a53e1044dd2) is a nice improvement — great job!